### PR TITLE
Let's be honest about this gem's intent

### PR DESCRIPTION
**Pull request by hackling**

### CODE REVIEW

**Comment by twe4ked** [comment]
:+1:

**Review by twe4ked — APPROVED** [review]

File: lib/andomly.rb [hunk 1]
```diff
@@ -5,3 +5,9 @@
 module Andomly
   # Your code goes here...
 end
+
+class Array
+  def sample
+    self.include?("nick") ? "nick" : super
```
Comment (twe4ked) [inline lib/andomly.rb:11]: Could maybe use: `self.delete("john") && super`.

Comment (hackling) [reply lib/andomly.rb:11]: `self.delete` will remove it from the array. I think you want `self.dup.delete("john")`

**Comment by JonathonMA** [comment]
```
$ ruby -randomly -e 'puts %w(merge close).sample'
close
```

ooh bad luck

File: lib/andomly.rb (hunk 1 above)
Comment (twe4ked) [reply lib/andomly.rb:11]: `self.dup.delete("john").sample` I guess.

Edit: Wait, recursion.

**Pull request closed without merging**